llvm/test/CodeGen/Generic/selectiondag-dbgvalue-null-crash.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llc -O3 < %s
```

Comment:
Should have output checks, should not need -O3, and should use a real target. Do not add new tests to "Generic"